internal/helpers/embed_file_system.go

```diff
@@ -67,7 +69,18 @@ func GenerateFilesByTemplateFromEmbedFS(efs embed.FS, templates []EmbedTemplate)
 
 		// Rename temp file.
 		if err := os.Rename(file.Name(), t.OutputFile); err != nil {
-			return err
+			// Define a new variable for the exit error.
+			var exitErr *os.LinkError
+
+			// Check, if the current error is exit error.
+			if errors.As(err, &exitErr) {
+				// Try to rename file in non-Unix system.
+				if err := renameFileNonUnix(file.Name(), t.OutputFile); err != nil {
```

Comment:
@afasari look this line, please.
Can you check with this code on your system?

Reply:
Yes, this works fine

So the first issue about cross-device link is already solved
but the second issue is still open

here is the results
![image](https://github.com/gowebly/gowebly/assets/24645546/af6c4b47-802f-465d-8925-1d1bb22c3e4d)

and when I debug, the root cause is because the package.json is an empty file instead of an empty JSON bracket, so the error npm install https://github.com/gowebly/gowebly/pull/3/files#r1343391066

Reply:
Yeah, I see what this is about. Your code that renames the file does not include data from embed FS.

That's why you get an error like in your screenshot.